code/BTCUtil.c

```diff
@@ -780,6 +780,17 @@ Datas ckdIndicesFromPath(const char *cStr)
     return DTPopDatas(result);
 }
 
+Data privKeyToHdWallet(Data privKey, const char *passphrase)
+{
+    Data seedPhrase = DataCopy("Bitcoin seed", strlen("Bitcoin seed"));
+
+    Data seed = PBKDF2(toMnemonic(privKey).bytes, passphrase);
+
+    Data hash = hmacSha512(seedPhrase, seed);
+
```

Comment:
After the assert, add another line for safety when asserts are disabled:
`if(hash.length != 64)
return DTPopNull();`